pytac/lattice.py

```diff
@@ -349,6 +326,65 @@ def set_element_values(self, family, field, values):
         if len(elements) != len(values):
             raise LatticeException("Number of elements in given array must be"
                                    " equal to the number of elements in the "
-                                   "family")
+                                   "family.")
         for element, value in zip(elements, values):
-            element.set_value(field, value)
+            element.set_value(field, value, handle=pytac.SP)
+
+    def set_default_arguments(self, default_units=None,
```

Comment:
It now looks a bit strange for this to be one method - I suggest you split it into two.